### ISSUE TITLE
lazydata command not recognizable on Windows

### ISSUE THREAD

**Issue by lastmeta** [rstojnic/lazydata#9]
> Add to your project
> To enable lazydata, run in project root:
> 
> $ lazydata init 

This resulted in:

> 'lazydata' is not recognized as an internal or external command, operable program or batch file.

on windows 10.

**Comment by lastmeta** [comment]
`python -m lazydata init`

> C:\ProgramData\Anaconda3\python.exe: No module named lazydata.__main__; 'lazydata' is a package and cannot be directly executed

**Comment by rstojnic** [comment]
I don't have an access to a Windows machine to test this. This is the content of the `lazydata/bin/lazydata` executable for Mac/Linux:

```
#!/usr/bin/env python
from lazydata.cli.cli import cli

if __name__ == "__main__":
    cli()
```

So, my guess is that one needs to add `lazydata/bin/lazydata.bat` that sets the pythonpath to include the lazydata root and executes the script above.

**Comment by lastmeta** [comment]
thanks, I think I fixed it, but I'm only testing it on windows so idk if I broke it on linux, anyway, this is what I did

[fork](https://github.com/LegitStack/lazydata/commit/0907c1f9d572c28d193524fef8e580309a7f8e15)

**Comment by rstojnic** [comment]
Looks like it's not breaking anything on a Mac, but would also like to run it through circleci. Would you mind creating a PR?

**Comment by lastmeta** [comment]
sure, I've sent it in.

**Comment by rstojnic** [comment]
Solved in #11
